linux/CMakeLists.txt

```diff
@@ -1 +1,7 @@
+install(FILES org.qgis.qgis.desktop DESTINATION share/applications)
 install(FILES org.qgis.qgis.appdata.xml DESTINATION share/metainfo)
+install(FILES ${CMAKE_SOURCE_DIR}/images/icons/qgis_icon.svg DESTINATION share/icons/hicolor/scalable/apps)
```

Comment:
Destination should be just `qgis.svg` or the `.desktop` must be updated accordingly